src/main/resources/migrations/00033_create_table_metadata_key.sql

```diff
@@ -0,0 +1,8 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:create_table_metadata_key
+CREATE TABLE metadata_key (
+    id BIGSERIAL PRIMARY KEY,
+    key VARCHAR(50) NOT NULL
```

Comment:
Is there anywhere we enforce the 50 character limit anywhere else? I'm guessing nothing will ever get this long but just in case!

Reply:
I think both connector and publicapi enforce it to max of 30 characters. If we ever change these limits, ledger should be updated too